src/components/SearchHeader/SearchHeader.tsx

```diff
@@ -5,60 +5,67 @@ import styles from './SearchHeader.module.css';
 import darkLogo from '../../images/dk-dark-logo.png';
 
 const SearchHeader: React.FunctionComponent = () => {
```

Comment:
The `SearchHeader` and `SearchPageHeader` are a bit confusing. Also they seem to serve similar purposes? Maybe we should just merge the two components.

Reply:
Initially I wanted them to be the same but I realized that one header should link to another page whilst the main SearchPageHeader just runs the search API again so it might be better to keep them separate.

Reply:
Just use a boolean in `props` to differentiate between the two?